src/gui/folderstatusmodel.cpp

```diff
@@ -157,10 +158,12 @@ QVariant FolderStatusModel::data(const QModelIndex &index, int role) const
         if (role == FolderStatusDelegate::AddButton) {
             return true;
         } else if (role == Qt::ToolTipRole) {
-            if (!_accountState->isConnected()) {
-                return tr("You need to be connected to add a folder");
+            if (_accountState) {
+                if (_accountState && !_accountState->isConnected()) {
+                    return tr("You need to be connected to add a folder");
+                }
```

Comment:
No need to check twice

```suggestion
                if (!_accountState->isConnected()) {
                    return tr("You need to be connected to add a folder");
                }
```

Reply:
sorry, it's a typo

Reply:
fixed